.gitignore

```diff
@@ -4,3 +4,4 @@
 /.gradle
 .DS_store
 /repo
+gradle.properties
```

Comment:
fyi, you can also specify the upload keys in the ~/.gradle/gradle.properties instead of gitignoring this (doesn't matter either way here)